packages/dapp/src/components/Web3AuthWrapper.tsx

```diff
@@ -78,6 +85,20 @@ export const Web3AuthWrapper: React.FunctionComponent = () => {
     return civilContext.auth.handleInitialState();
   }
 
+  React.useEffect(() => {
+    (window as any).context = civilContext; // @TODO/tobek tmp
+    civilContext.auth.setShowWeb3Login(handleLoginClicked);
+    civilContext.auth.setShowWeb3Signup(handleSignUpClicked);
+  }, [dispatch]);
+
```

Comment:
Can you also add log out?

Reply:
Good call, just went looking for it and found that it's actually already there at `context.auth.logout()`, set by `AuthService`.